best_practices.rst

```diff
@@ -88,7 +88,7 @@ application behavior.
 :ref:`Use env vars in your project <config-env-vars>` to define these options
 and create multiple ``.env`` files to :ref:`configure env vars per environment <config-dot-env>`.
 
-Use Secret for Sensitive Information
+Use Secrets for Sensitive Information
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Please also increase the length of the underline

Reply:
@javiereguiluz we also need to keep the old link/deeplink working, can you please share an example, I am currently on a phone